hasher-matcher-actioner/hmalib/scripts/populate_config_db.py

```diff
@@ -0,0 +1,142 @@
+#! /usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+# type: ignore
+
+"""
+Write simple configs to the HMA config database
+"""
+
+import argparse
+from dataclasses import fields
+import getpass
+import json
+import tempfile
+import subprocess
+import typing as t
+import re
+
+from hmalib.lambdas.fetcher import ThreatExchangeConfig
+from hmalib.common import config as hmaconfig
+
+SUPPORTED_CONFIGS = [
+    ThreatExchangeConfig,
+]
+
+
+def better_bool_type(s: str):
+    s = s.strip().lower()
+    if s in ("true", "1"):
```

Comment:
Do you want to add "True" and "False" as well?

Reply:
Since users are pythonic?

Reply:
lower() handles this